README.md

```diff
@@ -87,6 +87,8 @@ Contributions to this list are wery welcome :dolphin: You can add links through
 - [Quorum-Splunk](https://www.splunk.com/en_us/blog/it/introducing-splunk-app-for-quorum.html) - Using Splunk for quorum blockchain analytics and dashboards
 - [Mirror Besu](https://www.npmjs.com/package/mirror-besu) - compiling, testing and deployment for Hyperledger Besu, aiming to make life as an Enterprise Ethereum developer easier.
 - [Ethernal](https://tryethernal.com/) - Block explorer for private EVM-based chains.
+- [Quorum-toolbox](https://github.com/chainstack/quorum-toolbox) - Tools and scripts for Quorum configuration.
+- [Web3Quorum](https://github.com/chainstack/web3quorum) - A web3py extension that supports Quorum APIs: Raft and Istanbul.
```

Comment:
Should this be listed under section libraries next to other web3 libraries? Thanks!

Reply:
Sure, done.